other/bootstrap_daemon/src/config.c

```diff
@@ -156,7 +156,7 @@ int get_general_config(const char *cfg_file_path, char **pid_file_path, char **k
     }
 
     // Get PID file location
-    const char *tmp_pid_file;
+    const char *tmp_pid_file = NULL;
```

Comment:
`nullptr`? We use `nullptr` elsewhere in this file.

Reply:
Done.